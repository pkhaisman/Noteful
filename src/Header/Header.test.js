// import React from 'react';
// import ReactDOM from 'react-dom';
// import AddFolder from './AddFolder';
// import renderer from 'react-test-renderer';

// describe('AddFolder Component', () => {
//     it('renders without crashing', () => {
//       // create a DOM element to render the component into
//       const div = document.createElement('div');
//       //render the component, this is the actual test, if something is wrong it will fail here
//       ReactDOM.render(<AddFolder />, div);
//       //clean up code
//       ReactDOM.unmountComponentAtNode(div);
//     });
    
//     it('renders the UI as expected', () => {
//         const tree = renderer
//             .create(<AddFolder />)
//             .toJSON();
//         expect(tree).toMatchSnapshot();  
//     });
// });
