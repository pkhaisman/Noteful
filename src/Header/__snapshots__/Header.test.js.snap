// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders the UI as expected 1`] = `
<p>
  Header
</p>
`;
